<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signUp</class>
 <widget class="QWidget" name="signUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>457</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>457</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>457</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>1</y>
     <width>291</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/girl1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="helloLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Hello!</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>17</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 17pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Welcome To Chat</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>223</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="usernameTitleLabel">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>21</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 50 10pt &quot;Arial Black&quot;;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userNameLineEdit">
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="passwordTitleLabel">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>21</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 50 10pt &quot;Arial Black&quot;;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="confirmPasswordTitleLabel">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>21</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 50 8pt &quot;Arial Black&quot;;</string>
      </property>
      <property name="text">
       <string>Ensure your Password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="confirmPasswordLineEdit">
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="commitPushButton">
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 123, 255);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;</string>
      </property>
      <property name="text">
       <string>Sure</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="returnPushButton">
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 123, 255);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;</string>
      </property>
      <property name="text">
       <string>CLick there to Return Login</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
